documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1892,6 +1892,18 @@ See the {link-prefix}:{link-postgresql-connector}#postgresql-data-types[list of
 Each item (regular expression) must match the fully-qualified `<fully-qualified table>:<a comma-separated list of columns>` representing the custom key. +
 Fully-qualified tables could be defined as _schemaName_._tableName_.
 
+|[[postgresql-publication-autocreate-mode]]<<postgresql-publication-autocreate-mode, `publication.autocreate.mode`>>
+|_all_tables_
+|Determine how creation of a publication should work, the default is all_tables.
```

Comment:
Could you clarify that this option only applies when pgoutput is used? Also, perhaps a link to upstream Postgres docs on the concept of publications?